finetune/base.py

```diff
@@ -227,10 +224,7 @@ def finetune(self, Xs, Y=None, batch_size=None):
             # Validation with all other tasks.
             train_hooks.append(
                 tf.estimator.experimental.InMemoryEvaluatorHook(
-                    estimator,
-                    val_input_fn,
-                    every_n_iter=val_interval,
-                    steps=val_size // batch_size,
+                    estimator, val_input_fn, every_n_iter=val_interval, steps=val_size // batch_size
```

Comment:
Think we might need math.ceil here instead to prevent error when steps=0?

Reply:
I think this should also appear after a rebase. But i need to check.